packages/ui/src/components/Checkbox/style.less

```diff
@@ -125,38 +69,49 @@
     }
   }
 
-  &:hover > .condo-checkbox > .condo-checkbox-inner {
-    border-color: @condo-global-color-gray-7;
+  &-disabled .condo-checkbox-inner {
+    opacity: 0.5;
   }
 
-  &:hover > .condo-checkbox-checked > .condo-checkbox-inner::before {
-    border-radius: @checkbox-inner-border-radius;
-  }
+  .condo-checkbox.condo-checkbox-indeterminate > .condo-checkbox-inner {
+    background: @condo-global-color-brand-gradient-1;
 
-  .condo-checkbox-indeterminate > .condo-checkbox-input:focus + .condo-checkbox-inner::before {
-    .condo-checkbox-inner-layer(@condo-global-color-brand-gradient-1, @checkbox-inner-border-radius);
+    &::after {
+      width: 14px;
+      height: @condo-global-spacing-4;
+      background: @condo-global-color-brand-gradient-5 border-box;
+      border-radius: @condo-global-border-width-default;
+    }
   }
 
-  &:hover > .condo-checkbox-checked > .condo-checkbox-inner {
-    background: @condo-global-color-brand-gradient-5 border-box;
-    border-color: transparent;
+  .condo-checkbox.condo-checkbox-checked > .condo-checkbox-inner {
+    background: @condo-global-color-brand-gradient-1;
+
+    &::after {
+      top: @condo-checkbox-checkmark-top - 1;
+      left: @condo-checkbox-checkmark-left;
+      box-sizing: content-box;
+      width: @condo-checkbox-checkmark-width;
+      height: @condo-checkbox-checkmark-height;
+      border-width: 3px;
+      border-image-source: linear-gradient(45deg, #4cd174 0%, #6db8f2 100%);
+      border-image-slice: 1;
+      transform: rotate(48deg) scale(1) translate(-50%, -50%);
```

Comment:
why 48? and -1 above

Reply:
This is an already existing style, I did not change it)

Reply:
There is no need to change the degrees, they affect the tilt of checkmark